README.md

```diff
@@ -166,16 +166,47 @@ you can run ``ansible-playbook`` with the ``--connection=local`` flag.
 **Step 4**, become root and launch ``ansible-playbook`` locally:
 
     sudo -s  # become root, needs sudo password
-    ansible-playbook -i "localhost," --connection local playbook.yml
+    ansible-playbook -i "localhost," --connection=local playbook.yml
 
 When the playbook is done running, if you got no errors you can login at:
 
     https://localhost/admin
     username: admin
-    passowrd: admin
+    password: admin
 
 **Note**: do not use this method in a production system.
 
+Enabling the network topology module
+------------------------------------
+
+For enabling the network topology module you need to change the `openwisp2_network_topology` role variable
```

Comment:
To enable the network topology module you need to set  `openwisp2_network_topology` to `true `.